ingesters/http.md

```diff
@@ -592,3 +592,31 @@ Health-Check-URL="/logbot/are/you/alive"
 The HEC data format expects the timestamps to be Unix timestamps in seconds formatted as an integer or optionally a floating point number; the limited documentation also states that the timestamp should be encoded in the JSON object as a number.  The Gravwell HEC compatible ingester will make a best effort to extract and process timestamps generated by data providers that communicate with HEC endpoints However, if a timestamp cannot be interpreted at all, the endpoint will return a 400 response rather than attempt to process a potentially invalid timestamp.
 
 The HEC compatible endpoint contains special processing logic for timestamps where any Unix timestamps that are approximately the zero time will be treated as "NOW" and given the timestamp of ingest.  For example, a HEC event with the timestamp of `"time": 42` will be assigned the current UTC time.
+
+## Dynamic Configuration Reload
+
+The HTTP ingesters supports dynamic configuration reloading for many configuration items; the ingester can dynamically reload Listener, HEC, and Amazon Firehose listener configuration blocks without restarting.  The Ingester uses the `SIGHUP` system signal as a notification mechanism to reload its configuration.
+
+Upon successful reload of a configuration the ingester will emit a log to the `gravwell` tag with `loaded new config` in the `Message` portion.
+
+```
+<14>1 2025-10-28T20:21:03.459615Z 8aa069286fc3 httpingester - HttpIngester/main.go:170 - loaded new config
+```
+
+If the ingester cannot load the configuration set due to errors in the config, it will emit an error to the `gravwell` tag with `failed to parse new configuration` in the message body and continue using the same config.  Attempting to hot reload a broken config will NOT break the running ingester.
```

Comment:
```suggestion
If the ingester cannot load the configuration set due to errors in the config, it will emit an error to the `gravwell` tag with `failed to parse new configuration` in the message body and continue using the same config. Attempting to hot reload a broken config will NOT break the running ingester.
```